### PR TITLE
QA round 1 updates to Wildfire

### PR DESCRIPTION
**Pull request by clhenrick**
- [x] fix value of `imperial` in API calls
- [x] change location isn't updating in Chart title
- [x] add no data warning message to map view
- [ ] ~~values for decadal probability go over 1~~ (can't reproduce)
- [x] learn more copy for indicator
- [x] update no data color on map
- [x] make sure place names are visible on map
- [x] update default location for chart view

### CODE REVIEW

**Comment by clhenrick** [comment]
[Tools: Widlfire](https://trello.com/c/i33A7gwb/49-tools-widlfire)

File: src/routes/tools/wildfire/_ExploreData.svelte [hunk 1]
```diff
@@ -97,17 +98,24 @@
...
     noData = Math.max(...$dataStore.map((d) => d.values.length)) === 0;
   } else {
-    statsData = null;
     dataByDate = null;
   }
 
   afterUpdate(() => {
-    showMissingDataMsg = $locationStore.boundaryId !== "locagrid";
-    showNoDataMsg = $locationStore.boundaryId === "locagrid" && noData;
+    if (!activeTab) {
```
Comment (mukhtyar) [inline src/routes/tools/wildfire/_ExploreData.svelte:108]: yikes. the wildfire has just been a painful dataset to work with.

**Review by mukhtyar — APPROVED** [review]
Looks good to me.

File: src/routes/tools/wildfire/_ExploreData.svelte (hunk 1 above)
Comment (clhenrick) [reply src/routes/tools/wildfire/_ExploreData.svelte:108]: Yeah, it's why I've been talking with Brian about at least showing the % of grid cells with no data values in a boundary area.

**Pull request merged**